include/openssl/err.h

```diff
@@ -270,6 +257,8 @@ int ERR_load_ERR_strings(void);
 
 DEPRECATEDIN_1_1_0(void ERR_remove_thread_state(void *))
 DEPRECATEDIN_1_0_0(void ERR_remove_state(unsigned long pid))
+
+typedef struct err_state_st ERR_STATE;
 ERR_STATE *ERR_get_state(void);
```

Comment:
`ERR_get_state` is no longer usable with `ERR_STATE` made opaque. Perhaps just remove it? Debian code search comes up with only the following uses:

1. https://sources.debian.org/src/python2.7/2.7.16-2/Modules/_ssl.c/?hl=595#L595
2. https://sources.debian.org/src/python3.7/3.7.3-2/Modules/_ssl.c/?hl=909#L909
3. https://sources.debian.org/src/myproxy/6.2.4-1/ssl_utils.c/?hl=85#L85
4. https://sources.debian.org/src/canl-c/3.0.0-3/src/proxy/sslutils.c/?hl=404#L404

OpenSSL's blog post mentions, for 3.0, that "We plan to keep impacts on existing end user applications to an absolute minimum with the intention that the vast majority of existing well-behaved applications will just need to be recompiled". I think the "well-behaved" qualification covers all of these:

The calls in (1) and (2) are pointless and have already been [removed](https://github.com/python/cpython/commit/55e53c309359327e54eb74b101c5a3240ea9cd45) upstream.

(3) and (4) and would be broken by this CL anyway and so are not further negatively impacted by removing the function. (3) should just loop over `ERR_get_error` rather than reach into OpenSSL's internals. (4) is thoroughly not a well-behaved use of OpenSSL's APIs and would break anyway if OpenSSL were to allocate flag 64. (Also the flag it sets appears to be [unused](https://codesearch.debian.net/search?q=ERR_DISPLAY_CONTINUE_NEEDED)...)